### PR TITLE
Modify data prep for mSUPERB multilingual

### PR DESCRIPTION
**Pull request by guapaQAQ**

### CODE REVIEW

File: egs2/msuperb/asr1/conf/tuning/train_asr_fbank.yaml [hunk 1]
```diff
@@ -54,5 +54,5 @@ specaug_conf:
     num_time_mask: 10
 
 
-num_iters_per_epoch: 20000 # number of iterations per epoch
-max_epoch: 30   
\ No newline at end of file
+num_iters_per_epoch: 10000 # number of iterations per epoch
```
Comment (ftshijt) [inline egs2/msuperb/asr1/conf/tuning/train_asr_fbank.yaml:57]: Why change this to 10000, is there a specific reason?

File: egs2/msuperb/asr1/local/data.sh [hunk 2]
```diff
@@ -126,9 +126,7 @@ if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
     log "stage3: Create non-linguistic symbols for language ID"
     mkdir -p "$(dirname ${nlsyms_txt})"
     if "${multilingual}"; then
-        train_set=data/train_${duration}${suffix}
-        cut -f 2- ${train_set}/text | grep -o -P '\[.*?\]|\<.*?\>' | sort | uniq > ${nlsyms_txt}
-        log "save non-linguistic symbols in ${nlsyms_txt}"
+        log "no non-linguistic symbols needed"
```
Comment (ftshijt) [inline egs2/msuperb/asr1/local/data.sh:129]: Why do we disable this part?

File: egs2/msuperb/asr1/local/data_prep.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import argparse
 import os
+import string
```
Comment (ftshijt) [inline egs2/msuperb/asr1/local/data_prep.py:3]: You forgot to import re

Comment (guapaQAQ) [reply egs2/msuperb/asr1/local/data_prep.py:3]: Thanks

**Pull request merged**